tools/distrib/check_protobuf_pod_version.sh

```diff
@@ -22,6 +22,8 @@ pushd third_party/protobuf
 
 version1=$(git describe --tags | cut -f 1 -d'-')
 v1=${version1:1}
+# in case we got a 2-part "x.y" version number from the tag, convert it to version number in "3.x.y" format.
+v1=$(echo "$v1" | sed 's/^\([0-9]*\)\.\([0-9]*\)$/3.\1.\2/')
```

Comment:
Could you add a rationale on why the "3.x.y" format is chosen?

(i.e. maybe point to https://github.com/protocolbuffers/protobuf/releases if that's the reason?)

Reply:
done.

Reply:
Thanks so much for adding the detailed comments!